packages/graphql_flutter/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   meta: ^1.7.0
   path_provider: ^2.0.1
   path: ^1.8.0
-  connectivity_plus: ^6.0.1
+  connectivity_plus: ^6.0.3
```

Comment:
I am not super convinced about this, there is any reason to add this constraint?

if you want to use version 6.0.3 also with the current mainline version you should be able to do right?

Reply:
@vincenzopalazzo sorry for late reply. I'm using 

```
dependencies:
  graphql: ^5.2.0-beta.7
  graphql_flutter: ^5.2.0-beta.6
  connectivity_plus: ^6.0.3
```

but get a error when flutter pub get

```
Resolving dependencies... 
Because * depends on graphql_flutter ^5.2.0-beta.6 which depends on connectivity_plus ^5.0.0, connectivity_plus ^5.0.0 is required.
So, because * depends on connectivity_plus ^6.0.3, version solving failed.
```

<img width="1111" alt="Screenshot 2024-05-26 at 13 40 18" src="https://github.com/zino-hofmann/graphql-flutter/assets/20286370/d491ef3f-04c2-4a02-98c9-192e2cd69142">

and this is from pub_cache dir on my local

<img width="669" alt="Screenshot 2024-05-26 at 13 53 29" src="https://github.com/zino-hofmann/graphql-flutter/assets/20286370/4c10dbd2-7b62-4862-b90a-05aebd0c1108">

Maybe you don't release yet after this commit was merged

https://github.com/zino-hofmann/graphql-flutter/commit/fb0f708a540f545e7a329305a3311541011079fc

Reply:
Ah ok, `Because * depends on graphql_flutter ^5.2.0-beta.6 which depends on connectivity_plus ^5.0.0, connectivity_plus ^5.0.0 is required.` from this I do not think you need to bump the `connectivity_plus` again.

We already did in https://github.com/zino-hofmann/graphql-flutter/pull/1428 I just need to release the new beta

Can you revert this change and leave the `connectivity_plus: ^6.0.1`

Reply:
@vincenzopalazzo yes of course but I think 6.0.3 is better because it has some [fixes](https://pub.dev/packages/connectivity_plus/changelog).

Sooner or later we still need to upgrade it  because if app doesn't depend directly it will use by transitive. How do you about think?

Reply:
Make sense so, but this needs to be justified with another commit that links the changelog that you are now pointing to me

Reply:
I think these are 2 important commits:

v6.0.2: https://github.com/fluttercommunity/plus_plugins/pull/2763

v6.0.3: https://github.com/fluttercommunity/plus_plugins/pull/2836

Reply:
Yeah, divide your PR into two separate commits, wherein the first one you fix the crash and in the second one you upgrade the `connectivity_plus`

Reply:
@vincenzopalazzo I got it. Many thanks!

Reply:
@vincenzopalazzo Please check help me again!